bot/cogs/clickup.py

```diff
@@ -94,15 +96,20 @@ def __init__(self, bot: AutoShardedBot):
         result = await response.json()
 
         if "err" in result:
+            log.error("ClickUp responded to the task list request with an error!\n"
+                      f"error code: '{result['ECODE']}'\n"
+                      f"error: {result['err']}")
             embed.description = f"`{result['ECODE']}`: {result['err']}"
             embed.colour = Colour.red()
 
         else:
             tasks = result["tasks"]
 
             if not tasks:
-                embed.colour = Colour.red()
+                log.warning("{ctx.author} requested a list of ClickUp tasks, but no ClickUp tasks were found!")
```

Comment:
Incorrect level, having an empty task list is not an error. If a task list is actually empty then it's not an error or warning.

Reply:
yep, agreed